### ISSUE TITLE
Add filters to table view

### ISSUE THREAD

**Issue by skyripley** [Skyblueballykid/kiva-crowd#7]
After we've added filtering functionality to the `GET /loans` endpoint we can add filters to the frontend so that users can filter the table by supported fields.

Acceptance Criteria:
- Functioning filters added to table view
  - Should be built after #5  and #6

**Comment by skyripley** [comment]
Closing this as a "won't do" since this was more of a nice to have than a requirement. I think we'll be good to demo without this and if we want to show this functionality we can show the response in the swagger ui.
